src/signal_scope/plotwidget.cpp

```diff
@@ -104,6 +106,11 @@ PlotWidget::PlotWidget(PythonChannelSubscriberCollection* subscribers, QWidget *
   QTimer* labelUpdateTimer = new QTimer(this);
   this->connect(labelUpdateTimer, SIGNAL(timeout()), SLOT(updateSignalInfoLabel()));
   labelUpdateTimer->start(100);
+
+  rescalingTimer = new QTimer(this);
+  this->connect(rescalingTimer, SIGNAL(timeout()), SLOT(resetYAxisMaxScale()));
+  yRange[0] = FLT_MAX;
+  yRange[1] = FLT_MIN;
```

Comment:
Could you use:

std::numeric_limits<float>::max()
-std::numeric_limits<float>::max()

I think FLT_MIN is possibly a bug?  FLT_MIN is close to 0.0.